### ISSUE TITLE
Training on crops

### ISSUE THREAD

**Issue by dereyly** [open-mmlab/mmdetection#270]
Sorry for offtop theoretic question. 
Is training on crops working or not. I try to include in mmdetection crop training 3 times:
-- online random crops 
-- offline random crops
-- offline SNIPER crops
crop size is 512x512 that can be trained batch size 16 per GPU (x2 1080ti)
Reason I want to activate BatchNorm Training. But its not working for me. In MegDet and other works bigger batch size push accuracy with Sync Bn -- but with 128 GPU. 
In some competion speechs I heard that random crop is working. And it works in SNIPER -- i think it most impact of this paper how to work BN in crops.
Do you try to train on crops?

**Comment by hellock** [comment]
Yes training on crops would help a lot. Currently it is not implemented and we will post some updates if we have successfully tried it.

**Comment by marsggbo** [comment]
@dereyly Have you implemented training on crops?

**Comment by zvadaszi** [comment]
Any news on this feature? Thanks.

**Comment by dereyly** [comment]
I have implemented crops 
but accuracy drops -0.02 and -0.04 with SyncBN (8GPU, with bs=8x12)
Some experiments with 70 epochs , it is slightly better
I have visualization and in it everything is fine. 
Maybe need hard-negative mining. I do some experiments, but slow
